File: 3039/Brinquedos do Papai Noel.py
n = int(input())

meninos = 0
meninas = 0
while n!=0:
    nome, sexo = input().split()

    if sexo == "F":
        meninas+=1
    else:
        meninos+=1
   

    n-=1

print("%s carrinhos"%meninos)
print("%s bonecas"%meninas)

File: 3039/Brinquedos do Papai Noel.cpp
using namespace std;
#include <iostream>
#include <vector>
#include<bits/stdc++.h>
#include <algorithm>


int main(){

    int t, mo = 0, ma = 0;
    string nome, sexo;
    
    cin >> t; cin.ignore();
    
    while (t--){
        cin >> nome >> sexo;
        
        if (sexo == "F")
            ma++;
        else
            mo++;
    }
    
    cout << mo << " carrinhos" << endl;
    cout << ma << " bonecas" << endl;
    
    
}
